src/model/serialize_meta_details.rs

```diff
@@ -177,6 +185,7 @@ pub fn serialize_meta_details(
                             .iter()
                             .map(|stream| model::Stream {
                                 stream,
+                                progress: None,
```

Comment:
Can we track progress on trailers?
If they use the same streaming bucket I suppose we can?!

Reply:
We don't show a list of trailers in our apps we only play the first available one